jest.config.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  moduleNameMapper: {
+    "^vue$": "vue/dist/vue.common.js"
+  },
+  moduleFileExtensions: ["js", "json", "vue"],
+  transform: {
+    ".*\\.(vue)$": "vue-jest",
+    "^.+\\.js$": "babel-jest"
+  },
+  collectCoverage: true,
+  collectCoverageFrom: [
+    "lib/**/*.js",
+    "public/js/**/*.{js,vue}",
+    "!public/js/dist/**",
+    "!**/node_modules/**"
+  ],
+  roots: ["lib", "public/js"],
+  setupFiles: ["./lib/testSetup.js"],
+  testURL: "http://localhost/",
+  husky: {
```

Comment:
The `husky` option isn't part of the jest config and needs to remain in `package.json`. This is what's causing the Validation Warning when you run `npm t`.

Reply:
Oh interesting, I was wondering why that warning was coming up. Looks like master currently has the husky config nested inside of the jest flag int he package.json so that's why I ported it over to the new jest.config.js file 😅  : https://github.com/noopkat/electric-io/blob/master/package.json#L72-L104

Updating 😄